### ISSUE TITLE
Courses not visible while logged in as pupil in local dev

### ISSUE THREAD

**Issue by kevinbischof** [corona-school/backend#119]
There are no courses visible for the two dummy pupils Max and Tom.
In the psql db are no entries in table "subcourse_participants_pupil". 
Adding entries solves the problem in that the course is visible under "Your courses", but no courses are visible under "All courses".
There are also some mistakes with the assignment of subcourses and lectures.

**Comment by geroembser** [comment]
I think the problem is the following: the dev courses written when launching the web part of the backend locally for development are in many cases courses that are no longer joinable (and thus not shown in the frontend). 

For example, take the lectures of `subcourse2`: 
https://github.com/corona-school/backend/blob/0330e89cb4c27d8bacf3f6a653659346244903e3/web/dev.ts#L512
https://github.com/corona-school/backend/blob/0330e89cb4c27d8bacf3f6a653659346244903e3/web/dev.ts#L518
But the definition of `day` is the following: 
https://github.com/corona-school/backend/blob/0330e89cb4c27d8bacf3f6a653659346244903e3/web/dev.ts#L495
So this should probably be `getDate()` to make this work even for higher dates, like today (28). 

So in conclusion this is caused by inappropriate start time specifications of the lectures. 
In the frontend a course is only visible if it is joinable (i.e. its lectures are in the future or there is at least one lecture in the future while the course allows late joins)

**Comment by kevinbischof** [comment]
Thanks for explaining. I corrected the wrong usage of getDay and fixed a wrong push of a subcourse. Now it should be fine.
